xwiki-platform-core/xwiki-platform-web/src/main/webapp/templates/admin.vm

```diff
@@ -82,19 +82,8 @@
     ## notifiers) and we want to exclude them also.
     #if ($documentCount < 6)
       ## Less than 6 pages in the wiki, probably a fresh install.
-      #set ($URLhost = 'http://maven.xwiki.org')
-      #set ($URLgroup = 'org/xwiki/enterprise')
-      #set ($URLartifact = 'xwiki-enterprise-ui-mainwiki-all')
-      #if ("$!services.wiki" != '' && $services.wiki.currentWikiId != $services.wiki.mainWikiId)
-        #set ($URLartifact = 'xwiki-enterprise-ui-wiki-all')
-      #end
-      #set ($URLversion = $xwiki.getVersion())
-      #if ($URLversion.endsWith('-SNAPSHOT'))
-        #set ($URL = "${URLhost}/snapshots/${URLgroup}/${URLartifact}/$URLversion/")
-      #else
-        #set ($URL = "${URLhost}/releases/${URLgroup}/${URLartifact}/$URLversion/${URLartifact}-${URLversion}.xar")
-      #end
-      #warning($services.localization.render('admin.defaultwikinotinstalled_useflavor'))
+      #set ($URL = "https://www.xwiki.org/xwiki/bin/view/Documentation/AdminGuide/What%20to%20do%20with%20an%20empty%20XWiki/")
```

Comment:
This URL is still too hard to read. Could you rename it to `...AdminGuide/EmptyWiki` for ex?

Reply:
Note that the title of the page can remain :) And be "What to do with an empty wiki?"